modules/reference/pages/command/securityUtility-encode.adoc

```diff
@@ -55,7 +55,7 @@ securityUtility encode [options]
 |--encoding=_encoding_type_
 |Specifies how to encode the password.
 Supported encodings types are `xor`, `aes`, and `hash`.
-If this option isn't specified, a default encoding type of `xor` is used.
+If this option isn't specified, a default encoding type of `xor` is used. Currently, aes type uses AES-128 encryption.
```

Comment:
Currently, aes type uses AES-128 encryption
---->
 Currently, the `aes` type uses AES-128 encryption